.travis.yml

```diff
@@ -8,6 +8,7 @@ node_js:
   - '6'
   - '4'
 before_script:
+  - nvm install-latest-npm
```

Comment:
this needs to go in `before_install`, not `before_script`

Reply:
sorry!
I derped when I put it in and didn't read the suffix of the step....